utils/src/wizard/InputRevalidationStep.ts

```diff
@@ -0,0 +1,30 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AzureWizardPromptStep, IActionContext } from "../..";
+import { localize } from "../localize";
+
+export class InputRevalidationStep<T extends IActionContext> extends AzureWizardPromptStep<T> {
```

Comment:
This wasn't what I was expecting a step named `InputRevalidationStep` to do. Maybe `ConfirmPreviousInputStep` (or similar) would be a better name?